e2e/app/page/conceptset-actions-page.ts

```diff
@@ -30,7 +30,8 @@ export default class ConceptSetActionsPage extends AuthenticatedPage {
 
   async clickCreateAnotherConceptSetButton(): Promise<void> {
     const button = await this.getCreateAnotherConceptSetButton();
-    return button.clickAndWait();
+    await button.clickAndWait();
+    await waitWhileLoading(this.page);
```

Comment:
Flaky test failure happened during PR testing that requires immediate fix.

<img width="678" alt="Screen Shot 2021-02-24 at 6 55 27 PM" src="https://user-images.githubusercontent.com/35533885/109083027-18222500-76d3-11eb-987b-78249a04b65a.png">